core/tauri/src/api/process/command.rs

```diff
@@ -219,19 +219,12 @@ impl Command {
     self
   }
 
-  /// Sets the character encoding for stdout/stderr.
-  #[must_use]
-  pub fn encoding(mut self, encoding: &'static Encoding) -> Self {
-    self.encoding.replace(encoding);
-    self
-  }
-
   /// Spawns the command.
   ///
   /// # Examples
   ///
   /// ```rust,no_run
-  /// use tauri::api::process::{Command, CommandEvent};
+  /// use tauri::api::process::{Buffer, Command, CommandEvent};
```

Comment:
what's the use of this import?

Reply:
leftover from one of previous approaches, thanks for pointing it out